src/api/SFA.DAS.Provider.Events.Api/Web.config

```diff
@@ -21,25 +21,21 @@
     <add key="MonthEndConnectionString" value="server=.;database=DAS_PeriodEnd;trusted_connection=true;" />
     <add key="PaymentsV2ConnectionString" value="server=.;database=SFA.DAS.Payments.Database;trusted_connection=true;" />
     <add key="EventsConnectionString" value="server=.;database=DasPaymentsAT_Deds;trusted_connection=true;" />
-    <add key="LoggingRedisConnectionString" value="" />
-    <add key="LoggingRedisKey" value="" />
     <add key="ApiTokenSecret" value="dodge duck dip dive dodge" />
     <add key="ApiIssuers" value="http://localhost:57968/" />
     <add key="ApiAudiences" value="http://localhost" />
     <add key="MetadataEndpoint" value="MetadataEndpoint" />
     <add key="ConfigurationStorageConnectionString" value="UseDevelopmentServer=true" />
-    <add key="InstrumentationKey" value="na" />
+    <add key="APPLICATIONINSIGHTS_CONNECTION_STRING" value="InstrumentationKey=ae649c5f-3244-492e-a031-735721e58d4e;IngestionEndpoint=https://westeurope-1.in.applicationinsights.azure.com/;LiveEndpoint=https://westeurope.livediagnostics.monitor.azure.com/;ApplicationId=edfe3d88-37e5-4169-8e3b-2540f0f9a833" />

```

Comment:
We shouldn't be committing connection strings to public repos

From the developers channel:

Please ensure you are not committing sensitive data such as citizen data, plaintext passwords, private keys for certificates or API keys to your GitHub repositories. If you have inadvertently done so then please delete, expunge from your repository history and contact the security team immediately. :github-logo:

Reply:
Resolved by force pushing to remove commit from history